kge/job/train.py

```diff
@@ -490,6 +523,7 @@ def run_epoch(self) -> Dict[str, Any]:
         # add results to trace entry
         self.current_trace["epoch"].update(
             dict(
+                sum_loss=sum_loss,
```

Comment:
I think we should drop sum_loss again. Since we already report the right avg loss (averaged over number of examples), the sum_loss can be comkputed easiliy outside (num_examples is in the trace, right?). The good thing about only having avg loss is that is makes more clear that this is our training objective (which matters to interpret regulraizitaion constants)

Reply:
I Agree about clarity of training objective! Num_examples is not in the trace though, but will add it so sum loss can be computed from it